src/isar/apis/models/models.py

```diff
@@ -33,6 +35,13 @@ class RobotInfoResponse(BaseModel):
     plant_short_name: str
 
 
+@dataclass
```

Comment:
Should we consider making the `MediaConfig` response object a Pydantic Model by inheriting from `BaseModel`?

This could alternatively be deferred to a separate improvement issue.

Reply:
Yes, I think it might be easiest to do it in a separate issue https://github.com.mcas.ms/equinor/isar/issues/655